package.json

```diff
@@ -443,10 +503,28 @@
       ]
     },
     "views": {
-      "explorer": [
+      "truffle-activitybar-container": [
+        {
+					"id": "truffle-vscode.views.explorer",
+					"name": "Explorer"
+				},
         {
           "id": "truffle-vscode.truffle",
-          "name": "Truffle - Networks"
+          "name": "Networks",
+          "contextualTitle": "List of all known Networks"
+        },
+        {
+          "id": "truffle-vscode.views.deployments",
```

Comment:
Excluding this (and Libraries) for this upcoming release would be cool by me 👌

Reply:
got deployments working, have dropped libs